source/compatibility.tex

```diff
@@ -2004,6 +2004,35 @@
 };
 \end{codeblock}
 
+\diffref{class.copy.elision}
+\change
+A function returning an implicitly movable entity may invoke
+a constructor taking an rvalue reference to a type
+different from that of the returned expression.
+Function and catch-clause parameters can be thrown using move constructors.
+\rationale
+Side-effect of making it easier to write
+more efficient code that takes advantage of moves.
+\effect
+Valid C++17 code may become ill-formed in this International Standard.
+For example:
+\begin{codeblock}
```

Comment:
Use 2 space indentation please.

Reply:
Fixed.

Reply:
I'm not seeing this change.  Did you forget to push?

Reply:
It's there. Try "reload".

Reply:
my bad, sorry!